rust/agama-server/src/web/common.rs

```diff
@@ -17,6 +17,8 @@ use crate::error::Error;
 
 use super::Event;
 
+pub type Streams = Vec<(&'static str, Pin<Box<dyn Stream<Item = Event> + Send>>)>;
```

Comment:
I like this but, what about using `EventsStream` as a more descriptive name?

Reply:
in that case I would use `EventStreams` as it is not just single stream, but array of streams.

Reply:
Yes, you are right, that's even better.